tests/sqlparser_mssql.rs

```diff
@@ -448,6 +448,447 @@ fn parse_for_json_expect_ast() {
     );
 }
 
+#[test]
+fn parse_mssql_json_object() {
```

Comment:
noticed we used the parse_wildcard_expr, if that syntax is valid, can we include a test case with a wildcard argument?

Reply:
I used `parse_expr` instead.